src/main/java/com/exasol/projectkeeper/validators/pom/plugin/AbstractPluginPomValidator.java

```diff
@@ -193,13 +193,48 @@ private Node runXpathFromSegments(final Node plugin, final List<String> pathSegm
      */
     protected void verifyPluginPropertyHasExactValue(final Node plugin, final String propertyXpath,
             final Consumer<ValidationFinding> findingConsumer) {
-        if (verifyPluginHasProperty(plugin, propertyXpath, findingConsumer)) {
+        if (verifyPluginHasPropertyWithDefaultFromTemplate(plugin, propertyXpath, findingConsumer)) {
             final Node property = runXPath(plugin, propertyXpath);
             final Node templateProperty = runXPath(getPluginTemplate(), propertyXpath);
             validatePropertiesAreEqual(plugin, propertyXpath, findingConsumer, property, templateProperty);
         }
     }
 
+    protected void verifyPluginPropertyHasExactValue(final Node plugin, final String propertyXpath,
+            final Node expectedProperty, final Consumer<ValidationFinding> findingConsumer) {
+        if (verifyPluginHasPropertyWithGivenDefault(plugin, propertyXpath, expectedProperty, findingConsumer)) {
+            final Node property = runXPath(plugin, propertyXpath);
+            if (!isXmlEqual(property, expectedProperty)) {
+                findingConsumer.accept(ValidationFinding.withMessage(ExaError.messageBuilder("E-PK-71")
+                        .message("The {{plugin|uq}}'s configuration-property {{property path}} has an illegal value.",
+                                this.pluginArtifactId, propertyXpath)
```

Comment:
It would be helpful to add the actual value and the expected value to the message. That would make it easier to fix the issue.

Reply:
you're right. Done + added a test for it.